homeassistant/components/fritz/common.py

```diff
@@ -317,10 +321,7 @@ async def _async_update_data(self) -> UpdateCoordinatorDataType:
             await self.async_scan_devices()
             for key in list(self._entity_update_functions):
                 _LOGGER.debug("update entity %s", key)
-                entity_data["entity_states"][
-                    key
-                ] = await self.hass.async_add_executor_job(
```

Comment:
To my understanding, all the value functions just look up data in the coordinator, so there's no need to run them in executor threads?